packages/firestore/lite/api-extractor.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../config/api-extractor.json",
+  "mainEntryPointFilePath": "../dist/lite/firestore/lite/index.d.ts",
+  "dtsRollup": {
+    "enabled": true,
```

Comment:
Do you think you will ever need main and lite to share declarations? There is a risk where the same type from main and lite becomes incompatible if they use separate rollup d.ts files - see discussion [here](https://github.com/microsoft/rushstack/issues/1596#issuecomment-546790721). It's probably not a problem for us, since we are already using separate d.ts files for lite and main, but it's good to keep that in mind.